src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$days: 31;
+$month-length: 28, 29, 30, 31;
+$start-days: (
+  'mon': 0,
+  'tue': 101px,
+  'wed': 202px,
+  'thu': 303px,
+  'fri': 404px,
+  'sat': 505px,
+  'sun': 606px,
+);
+
+html {
+  height: 100%;
+}
+
 body {
+  height: 100%;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
```

Comment:
The width of `.calendar` is set to a hard-coded pixel value. This can lead to issues with responsiveness. Consider using a more responsive unit like percentages or viewport widths, or calculate the width based on the number of items and their widths plus the gap size.

Reply:
for this task we can't use relative width